src/components/cards/MyWallet/MyWallet.vue

```diff
@@ -53,22 +74,53 @@ const tokensWithBalance = computed(() => {
           class="font-semibold lg:font-normal ml-1 lg:ml-0"
           v-if="!isLoadingBalances"
         >
-          {{ etherBalance }} {{ nativeCurrency }}
+          <div
+            class="text-red-500 hover:text-red-700 dark:hover:text-red-400 transition-colors mr-0.5"
+            v-if="
+              Number(balanceFor(appNetworkConfig.nativeAsset.address)) === 0
```

Comment:
This can be replaced by first importing the `hasBalance` method from the `useTokens` composable in the script above:
```ts
const { hasBalance, nativeAsset } = useTokens();
```
Then in the template:
```suggestion
              !hasBalance(nativeAsset.address)
```